jbcli/jbcli/utils/dockerutil.py

```diff
@@ -192,47 +190,46 @@ def parse_dc_file(tag):
     :return: Full path to ECR image with tag.
     :rtype: ``string``
     """
+    if not os.path.isfile(os.getcwd() + '/docker-compose.yml'):
+        return
     base_ecr = '423681189101.dkr.ecr.us-east-1.amazonaws.com/'
     dc_list = []
-    if os.path.isfile(os.getcwd() + '/docker-compose.yml'):
-        with open("docker-compose.yml") as dc:
-            for line in dc:
-                if base_ecr in line:
-                    dc_list.append(line.split(':'))
-                    for pair in dc_list:
-                        pair = [i.strip().strip('\"') for i in pair]
+    with open("docker-compose.yml") as dc:
+        for line in dc:
+            if base_ecr in line:
+                dc_list.append(line.split(':'))
+                for pair in dc_list:
+                    pair = [i.strip().strip('\"') for i in pair]
 
-                        if 'controlcenter-dev' in pair[1]:
-                            full_path = base_ecr + 'controlcenter-dev:'
-                        elif 'juicebox-dev' in pair[1]:
-                            full_path = base_ecr + 'juicebox-devlandia:'
+                    if 'controlcenter-dev' in pair[1]:
+                        full_path = base_ecr + 'controlcenter-dev:'
+                    elif 'juicebox-dev' in pair[1]:
+                        full_path = base_ecr + 'juicebox-devlandia:'
 
-                        if tag is not None:
-                            full_path = full_path + tag
-                        else:
-                            full_path = full_path + pair[2]
-                        return full_path
+                    full_path = full_path + tag if tag is not None else full_path + pair[2]
```

Comment:
the precedence here is a little confusing, and I think the statement could be simplified anyway, to not repeat the `full_path +` part.

```suggestion
                    full_path = full_path + (tag if tag is not None else pair[2])
```